proto/api/v2/report_service.proto

```diff
@@ -265,4 +300,19 @@ service ReportService {
   rpc DeleteReport(DeleteReportRequest) returns (Empty) {
     option (google.api.http) = {delete: "/v2/reports/jobs/{id}/delete"};
   }
+
+  rpc PostViewBasedReport(ReportRequestViewBased) returns (RunReportResponseViewBased) {
+    option (google.api.http) = {
+      post: "/v2/reports/view-based/run"
+      body: "*"
+    };
+  }
+
+  rpc GetViewBasedReportMyHistory(ResourceByID) returns (ReportHistoryResponseViewBased) {
```

Comment:
What is the use of `ResourceByID` here? The API has no id. Should this be able to take in a query instead?